posts/2019-09-18-upcoming-docsrs-changes.md

```diff
@@ -4,7 +4,7 @@ title: "Upcoming docs.rs changes"
 author: The Rust Infrastructure Team
 ---
 
-On September 42nd breaking changes will be deployed to the [docs.rs] build
+On September 30nd breaking changes will be deployed to the [docs.rs] build
```

Comment:
Traditionally, it's 30**th**. 😉

Reply:
Did I mention I'm bad at multitasking during infra meetings?